### PR TITLE
Fix #269240: Add woodwinds and strings section to instruments list

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
See https://musescore.org/en/node/269198 and https://musescore.org/en/node/24091
Should go to master and 2.2

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
Hmm, I don't get what the Travis failure is about, Piano vs. Grand Piano, I didn't change anything there?

**Comment by lasconic** [comment]
Mmm... It has something to do with the reading of the instruments.xml file, the order of storing in QList and midi import... but I can't see why it doesn't work as expected.

File: share/instruments/instruments.xml [hunk 1]
```diff
@@ -10066,7 +10082,29 @@
...
+                  <Channel>
+                        <program value="48"/>
+                  </Channel>
+                  <Channel name="pizzicato">
+                        <program value="45"/>
+                  </Channel>
+                  <Channel name="tremolo">
+                        <program value="44"/>
+                  </Channel>
+                  <Instrument id="erhu">
```
Comment (Jojo-Schmitz) [inline share/instruments/instruments.xml]: Seem here is the error, not closing the prev. instrument

File: share/instruments/instruments.xml [hunk 2]
```diff
@@ -45,6 +45,22 @@
...
+                  <clef>G</clef>
+                  <bracket>1</bracket>
+                  <bracketSpan>2</bracketSpan>
+                  <barlineSpan>2</barlineSpan>
+                  <clef staff="2">F</clef>
+                  <aPitchRange>24-108</aPitchRange>
+                  <pPitchRange>24-108</pPitchRange>
+                  <Channel>
+                        <program value="73"/>
+                  </Channel>
```
Comment (Jojo-Schmitz) [inline share/instruments/instruments.xml]: same error here :-(

**Comment by Jojo-Schmitz** [comment]
I think I found and fix the mistake

**Comment by lasconic** [comment]
Ok. Here they are... back from the deads.

**Pull request merged**